### ISSUE TITLE
"d_loss_fake" and "d_loss_real" are not rigorously defined

### ISSUE THREAD

**Issue by bomb2peng** [yunjey/stargan#72]
Hi, 
In the solver.py, the two losses are defined as "torch.mean(out_src)", which are means of 16x1x2x2 tensors from the last conv layer output. These don't seem to be correct losses. Why not use cross entropy loss or MSE loss?
Best regards.

**Comment by talentCODE** [comment]
did you get the answer about d_loss_fake and d_loss_real. And the final feature 16*1*2*2 doesn't use sigmoid  activation
